CONTRIBUTING.md

```diff
@@ -154,6 +154,22 @@ Notable builds:
   runs Dokka's integration tests, which are designed to test compatibility with different Kotlin versions, with different
   multiplatform targets and with various user scenarios.
 
+### Gradle Build Scans
+
+[Gradle Build Scans](https://scans.gradle.com/) can provide insights into a Dokka Build. 
+JetBrains runs a Gradle Build Scan server https://ge.jetbrains.com/scans?search.rootProjectNames=dokka
+that can be used to automatically upload reports.
+
+To automatically opt in add the following to `$GRADLE_USER_HOME/gradle.properties`. 
```

Comment:
not sure, that suggesting to put those properties in `global` directory is good idea for both internal and external contributors :)
I would expect that it will be possible to put it into `local.properties` file in the project

Reply:
What's not a good idea about it? :) The property is prefixed with `org.jetbrains.dokka` so it shouldn't collide with other properties. It's a little inconvenient, but unfortunately Gradle declined support for local.properties https://github.com/gradle/gradle/issues/12283, so Dokka would have to add a custom implementation. I can do it, but I think it's more complication than its worth.

Reply:
The only thing that I don't like, is that `project-specific` properties are located in `global` place - so you can very easily forgot about them :)
But, it's a minor, not sure, that someone external will need to put them into `global` place.
At least for me, I would just put it into `gradle.properties` and add to separate `Change list` in IDEA, to not commit it accidentally.

Reply:
Yes, normally I would agree, but I think Build Scan is different and should be explicitly opted in, because it uploads identifiable information.

`kotlin.git` has a more complicated set up, where it logs a message to console and provides guidance on how to opt in. It then generates a local.properties file for Build Scan and other `kotlin.git` specific properties. I could recreate the same functionality, but I think that's a bit too complicated :)

Reply:
yeah, better to keep it simple